config.yml

```diff
@@ -17,6 +17,8 @@ PATHS:
   jsDir: "js"
   # Path to images folder
   imgDir: "images"
+  # Path to sprites folder
```

Comment:
Won't call it out for every line, but comments should end in a full stop punctuation.  Need to follow drupal standards for a successful d.o project.

Reply:
A lot of these were remnants from the original Foundation set-up. I'll go through and fix them.